packages/marketing-modal/src/Graphics/Graphics.tsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import { cx } from '@leafygreen-ui/emotion';
+import {
+  baseGraphicContainerStyle,
+  centeredGraphicContainerStyle,
+  filledGraphicContainerStyle,
+  baseGraphicStyle,
+  filledGraphicStyle,
+  curvedSVGBaseStyles,
+  curvedSVGThemeStyles,
+} from './styles';
+import { GraphicStyle } from '../MarketingModal/types';
+import { GraphicProps } from './types';
+import BlobSVG from '../BlobSVG/BlobSVG';
+
+/**
+ *
+ * @internal
+ */
+const Graphics = ({
+  graphic,
+  graphicStyle,
+  blobPosition,
+  showBlob,
+  theme,
+}: GraphicProps) => {
+  return (
+    <>
+      {showBlob && graphicStyle === GraphicStyle.Center && (
+        <BlobSVG blobPosition={blobPosition} theme={theme} />
+      )}
+      <div
+        className={cx(baseGraphicContainerStyle, {
+          [centeredGraphicContainerStyle]: graphicStyle === GraphicStyle.Center,
+          [filledGraphicContainerStyle]: graphicStyle === GraphicStyle.Fill,
+        })}
+      >
+        {React.cloneElement(graphic, {
+          className: `${graphic.props.className ?? ''} ${cx(baseGraphicStyle, {
+            [filledGraphicStyle]: graphicStyle === GraphicStyle.Fill,
```

Comment:
another nit since we keep performing this check we could pull it into a variable